packages/task/src/browser/task-configurations.ts

```diff
@@ -186,23 +192,29 @@ export class TaskConfigurations implements Disposable {
      * If reading a config file wasn't successful then does nothing.
      */
     protected async refreshTasks(configFileUri: string) {
-        const tasksConfigsArray = await this.readTasks(configFileUri);
-        if (tasksConfigsArray) {
+        const configuredTasksArray = await this.readTasks(configFileUri);
+        if (configuredTasksArray) {
             // only clear tasks map when successful at parsing the config file
             // this way we avoid clearing and re-filling it multiple times if the
             // user is editing the file in the auto-save mode, having momentarily
             // non-parsing JSON.
             this.removeTasks(configFileUri);
 
-            if (tasksConfigsArray.length > 0) {
+            if (configuredTasksArray.length > 0) {
                 const newTaskMap = new Map<string, TaskConfiguration>();
-                for (const task of tasksConfigsArray) {
+                for (const task of configuredTasksArray) {
                     newTaskMap.set(task.label, task);
                 }
                 const source = this.getSourceFolderFromConfigUri(configFileUri);
                 this.tasksMap.set(source, newTaskMap);
             }
         }
+
+        const cutomizations = await this.readTaskCustomizations(configFileUri);
```

Comment:
Typo: customisation

Reply:
ah i found something interesting: https://grammarist.com/spelling/customise-customize/

also do we want to align with spelling of customisation in other places such as 
https://github.com/theia-ide/theia/blob/551e49fc9988e3c2c7261d3d5f24177936e03c5b/packages/keymaps/src/browser/keybindings-widget.tsx#L147
?